x-pack/plugins/monitoring/public/services/clusters.js

```diff
@@ -69,9 +69,9 @@ export function monitoringClustersProvider($injector) {
       if (Legacy.shims.isCloud) {
         return Promise.resolve();
       }
-
+      const ccs = window.location.hash.split('?')[0] === '#/home' ? 'ccs' : 'single';
```

Comment:
Instead of this, why don't we just get the `globalState` and add `globalState.ccs` directly? The `prefixIndexPattern` code is designed to accept a `ccs` string like this